tests/page-objects/default/common/modal.wdio.page.js

```diff
@@ -20,16 +20,26 @@ const checkModalHasClosed = async () => {
   return await (await modal()).waitForDisplayed({ timeout: 5000, reverse: true });
 };
 
-const submit = async (timeout) => {
-  await (await submitButton()).waitForClickable({ timeout });
-  await (await submitButton()).click();
-  await checkModalHasClosed();
+const submit = async (timeout = 30000) => {
```

Comment:
The tests that failed for you are flaky unfortunately, and they we quite complicated to fix. 
I suggest you revert the changes that you made to fix those tests, and skick to just removing the references to horti. I will restart the tests until they pass.